docs/dashboard/home.md

```diff
@@ -60,4 +60,22 @@ If you prefer to work in dark mode, click on your account name in the top right
 
 ### How to log out
 
-To log out, click your account name in the top right corner of the dashboard, and then click the _log out_ button. 
+To log out, click your account name in the top right corner of the dashboard, and then click the _log out_ button.
+
+### How to use the dashboard navigator
+
+Click the navigator button on top right corner of the dashboard or use the *CTRL* + *K* combination to open the dashboard navigator. Then type the location you would like to navigate to, for instance: products main page.
+
+![Dashboard navigator](screenshots/dashboard-home-navigator.png)
+
+You can type "?" to see all available navigation options.
+
+Available navigation options:
```

Comment:
This description doesn't add any explanation on how to use it other than doubling text which user already can see.

Reply:
Ok, I will add an explanation on how to use it.